README.md

```diff
@@ -457,7 +533,7 @@ defmodule MyApp.UsersChannel do
   end
 
   def handle_in("ping", _payload, socket) do
-    user = Guardian.Channel.current_resource(socket)
+    user = current_resource(socket)
     broadcast socket, "pong", %{ message: "pong", from: user.email }
```

Comment:
What do you think about adding parenthesis and removing the extra whitespace in `%{}` for consistency?

```elixir
broadcast(socket, "pong", %{message: "pong", from: user.email})
```